### PR TITLE
Fix: parallelizable tests output is out of order (C# & gdscript)

### PR DESCRIPTION
**Pull request by OpalSoPL**
Fix for output of  tests when running parallelizable tests in verbose mode.

before the fix, results of test were scattered without any connection to its own source test class
Example: (tests from this repository)
before

After:
after

Tasks to do before merging
- [x] resolve conflicts
- [x] fix any errors after resolving conflicts

### CODE REVIEW

**Comment by MASSHUU12** [comment]
I was able to fix the conflicts. All that remained was to test the changes. The problems I found were tests written in GDScript displaying incorrectly in verbose mode, and for some reason ConfirmaBottomPanel displays incorrectly in the editor.

aaa

I'll look into these problems as I find a moment, if you don't address them yourself in the meantime.

By the way, I see correctly that you are using PowerShell? Strange that the colors don't display, that's something to look into if you'd be interested, if you haven't turned them off manually.

**Comment by OpalSoPL** [comment]
>All that remained was to test the changes. The problems I found were tests written in GDScript displaying incorrectly in verbose mode,

It might be related to testLog class. I'll look at it when I return to home

> By the way, I see correctly that you are using PowerShell? Strange that the colors don't display, that's something to look into if you'd be interested, if you haven't turned them off manually.

That was a text editor from cinnamon (Linux mint), I just saved output of test to txt file to compare each other. In terminal colours are present

**Comment by OpalSoPL** [comment]
obraz
 The panel looks normal, but something is weird with the test output. The logs are different from what I got before resolving the conflicts.

File: addons/confirma/src/classes/executors/CsTestExecutor.cs [hunk 1]
```diff
@@ -81,6 +82,10 @@ public int Execute(out TestResult? result)
 
     private void ExecuteClass(TestingClass testClass, TestResult result)
     {
+        List<TestLog> testLogs = new()
+        {
+            new(ELogType.Class, testClass.Type.Name)
+        };
         Log.Print($"> {testClass.Type.Name}...");
```
Comment (OpalSoPL) [inline addons/confirma/src/classes/executors/CsTestExecutor.cs:89]: That line is the reason for duplicated class name

Comment (OpalSoPL) [reply addons/confirma/src/classes/executors/CsTestExecutor.cs:89]: same thing occurs in GdTestExecutor

**Comment by OpalSoPL** [comment]
I fixed what was found and tested it, but found nothing else

**Review by MASSHUU12 — APPROVED** [review]
Everything looks in order. Thank you for this PR, I'll merge it right away.

**Pull request merged**